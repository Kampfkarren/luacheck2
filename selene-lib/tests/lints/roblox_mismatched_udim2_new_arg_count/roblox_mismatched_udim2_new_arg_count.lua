UDim2.new(1)
UDim2.new(1, 1)
UDim2.new(1, 1, 1)
UDim2.new(1, 1, 1, 1)
UDim2.fromOffset(1, 1)
UDim2.fromScale(1, 1)
