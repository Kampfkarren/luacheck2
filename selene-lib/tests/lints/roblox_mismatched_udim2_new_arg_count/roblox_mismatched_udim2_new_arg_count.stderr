error[roblox_mismatched_udim2_new_arg_count]: UDim2.new takes 4 numbers, but 1 were provided.
  ┌─ roblox_mismatched_udim2_new_arg_count.lua:1:1
  │
1 │ UDim2.new(1)
  │ ^^^^^^^^^^^^
  │
  = help: did you mean to use UDim2.fromScale or UDim2.fromOffset instead?

error[roblox_mismatched_udim2_new_arg_count]: UDim2.new takes 4 numbers, but 2 were provided.
  ┌─ roblox_mismatched_udim2_new_arg_count.lua:2:1
  │
2 │ UDim2.new(1, 1)
  │ ^^^^^^^^^^^^^^^
  │
  = help: did you mean to use UDim2.fromScale or UDim2.fromOffset instead?

error[roblox_mismatched_udim2_new_arg_count]: UDim2.new takes 4 numbers, but 3 were provided.
  ┌─ roblox_mismatched_udim2_new_arg_count.lua:3:1
  │
3 │ UDim2.new(1, 1, 1)
  │ ^^^^^^^^^^^^^^^^^^
  │
  = help: did you mean to use UDim2.fromScale or UDim2.fromOffset instead?

